### PR TITLE
Add initial tdd setup

### PR DESCRIPTION
**Pull request by iamfiropo**
Add initial tdd setup libraries i.e mocha, chai, chai-http

### CODE REVIEW

File: test/role.manager.spec.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import chaiHttp from 'chai-http';
+import chai, { expect } from 'chai';
+import { config } from 'dotenv';
+
+config();
+
+chai.use(chaiHttp);
```
Comment (thatkazuk1) [inline test/role.manager.spec.js]: Add a newline here @Johnpeace.

Use this setting in your VSCode `settings.json` file:
` "files.insertFinalNewline": true,`

Comment (iamfiropo) [reply test/role.manager.spec.js]: done

**Review by thatkazuk1 — CHANGES_REQUESTED** [review]
You have merge conflicts @Johnpeace. Can you please rebase before pushing again?

**Review by thatkazuk1 — APPROVED** [review]
Looks good to me

**Comment by thatkazuk1** [comment]
[Setup unit testing. Ensure all implemented functionality is tested henceforth.](https://trello.com/c/ScqaIdhg/5-setup-unit-testing-ensure-all-implemented-functionality-is-tested-henceforth)

**Pull request merged**